opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -1970,7 +1979,7 @@ void CpGrid::addLgrsUpdateLeafView(const std::vector<std::array<int,3>>& cells_p
             OPM_THROW_NOLOG(std::logic_error, "Adding LGRs to a distributed grid is not supported, yet.");
         }
     }
-    if (startIJK_vec.size() > 0) {
+    if (startIJK_vec.size() > 1) {
```

Comment:
Why dies it need to be 2 or larger?

Reply:
To have at least 2 blocks of cells to compare. If there is only one block of cells, it does not share any face with any other block since there is no other block. So, in that single block case, this check can be omitted.